eclair-core/src/main/resources/reference.conf

```diff
@@ -327,6 +327,20 @@ eclair {
         lock-timeout = 5 seconds // timeout for the lock statement on the lease table
         auto-release-at-shutdown = true // automatically release the lock when eclair is stopping
       }
+      safety-checks {
+        // a set of basic checks on data to make sure we use the correct database
+        enabled = false
+        max-age {
+          local-channels = 3 minutes
+          network-nodes = 30 minutes
+          audit-relayed = 10 minutes
```

Comment:
It's not very clear from this file alone what those refer to.
Can you add comments to each of them to help node operators properly configure it?

Reply:
Done with cd1905115.